docs/guides/unbound.md

```diff
@@ -52,9 +52,9 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-Optional: Download the list of primary root servers (serving the domain `.`). Unbound ships its own list but we can also download the most recent list and update it whenever we think it is a good idea. Note: there is no point in doing it more often then every 6 months.
+**Important**: Download the current list of primary root servers (serving the domain `.`). Updated it roughly every six. Note that this file changes infrequently.
```

Comment:
Update <> Updated 

Do we want to mention that this is called the 'root hints' file since that's the term they would need to search for more information?

Reply:
And I don't know if 'six' or '6' is the proper one, I'm sure someone else will find it.